### PR TITLE
Update automated updates workflow

### PR DESCRIPTION
**Pull request by jamesrweb**
## Proposed Changes

- Prefer PR generation action defaults and recommendations

### CODE REVIEW

File: .github/workflows/UPDATE_DEPENDENCIES.yml [hunk 1]
```diff
@@ -25,13 +25,5 @@ jobs:
       - name: Create dependency updates PR
         uses: peter-evans/create-pull-request@v4
         with:
-          token: ${{ secrets.GH_TOKEN }}
-          commit-message: Update project dependencies
```
Comment (jamesrweb) [inline .github/workflows/UPDATE_DEPENDENCIES.yml:29]: The token defaults to the `GITHUB_TOKEN`, commit-message, title and body also have some defaults in place. The delete-branch key allows us to delete the branch if the PR is closed or merged by the bot and the branch-suffix just creates unique branches per PR.

**Review by jamesrweb — COMMENTED** [review]
Please check [the action documentation](https://github.com/peter-evans/create-pull-request#action-inputs) before merging to be sure you don't want to add or remove anything else 👍🏻.

**Comment by yevdyko** [comment]
@jamesrweb Do we really need to run it daily since it can make a lot of noise if it creates minor PRs often? Is it for testing?

**Comment by jamesrweb** [comment]
> @jamesrweb Do we really need to run it daily since it can make a lot of noise if it creates minor PRs often? Is it for testing?

Exactly, for testing until we get it "right".

**Comment by yevdyko** [comment]
> > @jamesrweb Do we really need to run it daily since it can make a lot of noise if it creates minor PRs often? Is it for testing?
> 
> Exactly, for testing until we get it "right".

This makes sense. I also added `labels` and `team-reviewers`. Do we need to have a custom title since the default one "Changes by create-pull-request action" is not relevant to us?

**Comment by jamesrweb** [comment]
> > > @jamesrweb Do we really need to run it daily since it can make a lot of noise if it creates minor PRs often? Is it for testing?
> 
> > 
> 
> > Exactly, for testing until we get it "right".
> 
> 
> 
> This makes sense. I also added `labels` and `team-reviewers`. Do we need to have a custom title since the default one "Changes by create-pull-request action" is not relevant to us?

Sure, feel free to add a custom one to the review then.

For the team reviewers the only issue might be that the default GITHUB_TOKEN might not have the right permissions but only one way to find out!

**Comment by yevdyko** [comment]
> > > > @jamesrweb Do we really need to run it daily since it can make a lot of noise if it creates minor PRs often? Is it for testing?
> > 
> > 
> > > 
> > 
> > 
> > > Exactly, for testing until we get it "right".
> > 
> > 
> > This makes sense. I also added `labels` and `team-reviewers`. Do we need to have a custom title since the default one "Changes by create-pull-request action" is not relevant to us?
> 
> Sure, feel free to add a custom one to the review then.
> 
> For the team reviewers the only issue might be that the default GITHUB_TOKEN might not have the right permissions but only one way to find out!

Yes, we can check the reviewers tomorrow morning after the script runs, otherwise we will set our account names explicitly

**Review by yevdyko — APPROVED** [review]
@jamesrweb I think so far so good, the rest can be adjusted tomorrow after seeing the results.

**Pull request merged**